pkg/testfixtures/storage/mysql.go

```diff
@@ -125,12 +124,12 @@ func (m *mySQLTestContainer) RunMySQLTestContainer(t testing.TB) DatastoreTestCo
 	}
 
 	mySQLTestContainer := &mySQLTestContainer{
-		addr:     fmt.Sprintf("localhost:%s", p[0].HostPort),
+		addr:     "localhost:" + p[0].HostPort,
 		username: "root",
 		password: "secret",
 	}
 
-	uri := fmt.Sprintf("%s:%s@tcp(%s)/defaultdb?parseTime=true", mySQLTestContainer.username, mySQLTestContainer.password, mySQLTestContainer.addr)
+	uri := mySQLTestContainer.username + ":" + mySQLTestContainer.password + "@tcp(" + mySQLTestContainer.addr + ")/defaultdb?parseTime=true"
```

Comment:
This is arguably harder to read now. I'd vote that we keep this and the lines below as they were.

Reply:
Actually never mind that. Could you leave this as is and add `perfsprint` to `.golangci.yaml`?

Include
```go

linters-settings:
  perfsprint:
    errorf: false
```

Reply:
https://github.com/catenacyber/perfsprint/issues/40

Reply:
added in [`942b97b`](https://github.com/openfga/openfga/pull/2241/commits/942b97b8cdbab9d529abf725dc371951c88338db)